JSONAPI/Http/QueryableToManyRelatedResourceDocumentMaterializer.cs

```diff
@@ -46,13 +46,5 @@ public abstract class QueryableToManyRelatedResourceDocumentMaterializer<TRelate
         /// </summary>
         protected abstract Task<IQueryable<TRelated>> GetRelatedQuery(string primaryResourceId, CancellationToken cancellationToken);
 
-        /// <summary>
-        /// If the client doesn't request any sort expressions, these expressions will be used for sorting instead.
-        /// </summary>
-        /// <returns></returns>
-        protected virtual string[] GetDefaultSortExpressions()
```

Comment:
Ok, I supposed so just after deletation. I'll bring it back.

This one was not clear to me at the first glance... I'll bring it back too.
```C#
            if (sortExpressions == null || sortExpressions.Length < 1)
                sortExpressions = GetDefaultSortExpressions();`
```